test/featurizations/GraphNodeFeaturization_tests.jl

```diff
@@ -9,33 +9,43 @@ using ChemistryFeaturization.Featurization
     # custom lookup table...
     test_df = CSV.read(abspath(@__DIR__, "..", "test_data", "lookup_table.csv"), DataFrame)
 
-    @testset "Construct, encode, decode" begin
+    local fnames = ["X", "Block", "Atomic mass"]
+
+    @testset "Encode" begin
         # make sure both constructors give the same results
-        local fnames = ["X", "Block", "Atomic mass"]
         features = ElementFeatureDescriptor.(fnames)
         fzn1 = GraphNodeFeaturization(features)
         fzn2 = GraphNodeFeaturization(fnames)
 
         triangle_C_1 = AtomGraph(Float32.([0 1 1; 1 0 1; 1 1 0]), ["C", "C", "C"])
         triangle_C_2 = AtomGraph(Float32.([0 1 1; 1 0 1; 1 1 0]), ["C", "C", "C"])
-        # (this one also tests broadcasting of featurize!)
-        featurize!.([triangle_C_1, triangle_C_2], [fzn1, fzn2])
-        @test triangle_C_1.encoded_features == triangle_C_2.encoded_features
+        # @test fzn1 == fzn2
+
+        featurized_1, featurized_2 = featurize.([triangle_C_1, triangle_C_2], [fzn1]) # featurize can be broadcasted
```

Comment:
I think there's a way to overwrite the way broadcasts/iterators/whatever work to be able to do it even if it's not in brackets...

Reply:
There is?
I think `featurize.([triangle_C_1, triangle_C_2], fzn1)` throws an error.

Reply:
Yeah it definitely will right now. Damn, I think I asked about this exact thing on the Slack awhile ago and now I can't remember what the clean way to do it was...okay, well this shouldn't be a showstopper for now though I definitely do want to figure it out so another one for the project board haha